common/lib/common-definitions/src/logger.ts

```diff
@@ -59,6 +59,29 @@ export interface ITelemetryPerformanceEvent extends ITelemetryGenericEvent {
     duration?: number; // Duration of event (optional)
 }
 
+/**
+ * A property to be logged to telemetry containing both the value and the tag
```

Comment:
Let's put more info in this comment, in terms of documenting what "tag" means now that it's just a string

Reply:
Done